CHANGELOG.md

```diff
@@ -2,16 +2,22 @@
 
 ## Next (Unreleased)
 
+## v.0.2.6 (June 10th, 2024)
```

Comment:
Note: I didn't mention the dataset download because I don't want to advertise the feature until the Kaggle Cache resolver is implemented (in #133).